File: src/main/java/com/hoi4utils/clausewitz/data/country/PoliticalPower.java
package com.hoi4utils.clausewitz.data.country;

public class PoliticalPower {
    private int politicalPower;

    public PoliticalPower(int politicalPower) {
        this.politicalPower = politicalPower;
    }

    public int amt() {
        return politicalPower;
    }

    public void set(int politicalPower) {
        this.politicalPower = politicalPower;
    }
}

File: src/main/scala/com/hoi4utils/hoi4mod/history/countries/PoliticalPower.scala
package com.hoi4utils.hoi4mod.history.countries

class PoliticalPower(private var politicalPower: Int) {
  def amt: Int = politicalPower

  def set(politicalPower: Int): Unit = {
    this.politicalPower = politicalPower
  }
}

